### PR TITLE
Correcting minor typo in FAQ for --no-kubernetes

### PR DESCRIPTION
**Pull request by cchesser**
Minor correction on typo that I noticed when reading the FAQ.

### CODE REVIEW

File: site/content/en/docs/faq/_index.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ minikube start --driver=hyperv
 ```
 Alternatively, you can use `minikube image build`  instead of `minikube docker-env` and `docker build`.
 
-## Can I start minikube without Kuberentes runninng?
+## Can I start minikube without Kuberentes running?
```
Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:42]: can you swap Kuberentes to Kubernetes while we're here? thanks for catching this

Comment (cchesser) [reply site/content/en/docs/faq/_index.md]: Definitely: 0550d97966db2b70300cc25501c9c8fc56d83548

**Comment by sharifelgamal** [comment]
Thanks! We can merge this once you're signed the CLA.

**Comment by cchesser** [comment]
I have signed the CLA.

**Comment by cchesser** [comment]
Alright, CLA should be signed now - got email confirmation from HelloSign.

**Review by sharifelgamal — APPROVED** [review]
Thank you for your contribution!

**Comment by cchesser** [comment]
Thank you for the quick feedback!

**Pull request merged**